tests/foreman/installer/test_installer.py

```diff
@@ -1564,3 +1583,108 @@ def test_installer_timestamp_logs():
 
     :CaseAutomation: NotAutomated
     """
+
+
+@pytest.mark.stubbed
+@pytest.mark.tier3
+def test_positive_capsule_installer_and_register():
```

Comment:
We actually have methods that will do this integration, which would make for a short path to automating this test case:

https://github.com/SatelliteQE/robottelo/blob/master/robottelo/hosts.py#L815

https://github.com/SatelliteQE/robottelo/blob/master/robottelo/hosts.py#L31

Reply:
Thanks @mshriver, I will plan it in a separate MR.